index.rst

```diff
@@ -398,7 +398,7 @@ The spawner implementation will assume that the ``token`` element of the authent
 
 ``poll``
     Calls ``GET /nublado/spawner/v1/labs/<username>`` to see if the user has a running lab.
-    Returns ``None`` if the lab is in ``starting``, ``running``, or ``terminating`` state, and ``0`` if it is in ``failed`` state or does not exist.
+    Returns ``None`` if the lab is in ``pending``, ``running``, or ``terminating`` state, ``0`` if it does not exist, and ``2`` if the lab pod is in ``failed`` state.
```

Comment:
As long as this still lets people spawn a new lab. I was worried that reporting the failed state to JupyterHub would prevent the user from spawning a fresh lab.

Reply:
Right now, the UI eventually turns red and says "Spawn Failed" but you can go back to the spawner page and try again, which I think is the correct behavior.  The documentation matches the current behavior.  If we need to change it we will change both implementation and documentation.